userspace/libsinsp/parsers.cpp

```diff
@@ -724,7 +724,12 @@ bool sinsp_parser::reset(sinsp_evt *evt)
 		etype == PPME_SYSCALL_VFORK_17_X ||
 		etype == PPME_SYSCALL_VFORK_20_X ||
 		etype == PPME_SYSCALL_CLONE3_X ||
-		etype == PPME_SCHEDSWITCH_6_E)
+		etype == PPME_SCHEDSWITCH_6_E ||
+		/* If we received a `procexit` event it means that the process
+		 * is dead in the kernel, `query_os==true` would just generate fake entries.
+		 */
+		etype == PPME_PROCEXIT_E ||
+		etype == PPME_PROCEXIT_1_E)
```

Comment:
Is this chunk a bugfix that we could submit independent of the whole PR?

Reply:
uhm yes if you want we can do that...